components/selection/selection-action.js

```diff
@@ -7,24 +7,20 @@ import { ifDefined } from 'lit-html/directives/if-defined.js';
 import { LocalizeCoreElement } from '../../lang/localize-core-element.js';
 import { RtlMixin } from '../../mixins/rtl-mixin.js';
 import { SelectionInfo } from './selection-mixin.js';
-import { SelectionSubscriberMixin } from './selection-subscriber-mixin.js';
+import { SelectionInfoController } from './selection-info-controller.js';
 
 /**
  * An action associated with a selection component.
  * @fires d2l-selection-action-click - Dispatched when the user clicks the action; provides the selection info
  */
-class Action extends LocalizeCoreElement(SelectionSubscriberMixin(ButtonMixin(RtlMixin(LitElement)))) {
+class Action extends LocalizeCoreElement(ButtonMixin(RtlMixin(LitElement))) {
 
 	static get properties() {
 		return {
 			/**
 			 * Preset icon key (e.g. "tier1:gear")
 			 */
 			icon: { type: String, reflect: true },
-			/**
-			 * Indicates whether the action is only focusable
-			 */
-			nonInteractive: { type: Boolean, attribute: 'non-interactive', reflect: true },
```

Comment:
Again, wasn't quite sure why this needed to be a public thing that was set as opposed to just a computed internal thing based on `disabled` + `requires-selection`.

Reply:
I think this action component needs to expose this computed state, so that `d2l-overflow-group` can properly reflect the state in it's corresponding overflow menu-items.

Reply:
Hmmm... no I remember why I chose to use a new property - `disabled` is defined and provided by the `ButtonMixin` as a public property, and I wanted a property that did not conflict with that because the consumer could specify the button to be `disabled`, and we would not want to unset that when consumers select items.